tests/modules/keyspace_events.c

```diff
@@ -63,7 +65,14 @@ static int KeySpace_NotificationLoaded(RedisModuleCtx *ctx, int type, const char
 
 static int KeySpace_NotificationGeneric(RedisModuleCtx *ctx, int type, const char *event, RedisModuleString *key) {
     REDISMODULE_NOT_USED(type);
-
+    const char *key_str = RedisModule_StringPtrLen(key, NULL);
+    if (strncmp(key_str, "count_dels_", 11) == 0 && strcmp(event, "del") == 0) {
+        if (RedisModule_GetContextFlags(ctx) & REDISMODULE_CTX_FLAGS_MASTER) {
+            dels++;
+            RedisModule_Replicate(ctx, "keyspace.incr_dels", "");
```

Comment:
i'm confused.
don't we get both a `DEL` and an `incr_dels` on the replica?
so with your example, if we'll call `get_dels` on both servers we'll get a different result?

Reply:
No, on replica we increase the `dels` on on `incr_dels`. See that on the notification callback we check that we are master.